kafka-rest/src/test/java/io/confluent/kafkarest/integration/ConsumerTimeoutTest.java

```diff
@@ -30,10 +30,11 @@
   private static final Integer requestTimeout = 500;
```

Comment:
nit: On all of these I would add a suffix indicating that they are in millis, and would also use capitalized snake case (e.g. `REQUEST_TIMEOUT_MS`).